libraries/core_libs/network/src/network.cpp

```diff
@@ -110,10 +110,10 @@ void Network::setSyncStatePeriod(uint64_t period) { taraxa_capability_->setSyncS
 
 void Network::handleMaliciousSyncPeer(dev::p2p::NodeID const &id) { taraxa_capability_->handleMaliciousSyncPeer({id}); }
 
-void Network::onNewPbftVotes(std::vector<std::shared_ptr<Vote>> &&votes) {
+void Network::onNewPbftVotes(std::vector<std::shared_ptr<Vote>> &&votes, bool reward_votes) {
   for (auto &vote : votes) {
     LOG(log_tr_) << "Network broadcast PBFT vote: " << vote->getHash();
-    taraxa_capability_->onNewPbftVote(std::move(vote));
+    taraxa_capability_->onNewPbftVote(std::move(vote), reward_votes);
```

Comment:
btw why are we sending votes one by one and not in batches of max N votes ?

Reply:
I just use the current exist functions for votes sending.
Create a separate task for that https://github.com/Taraxa-project/taraxa-node/issues/1700

Reply:
@gaoqi8649 do you realize that you just created multiple conflicts because you separated these 2 PR's and you will have to resolve it later ? Why did you do it like this ?

Reply:
I remember we discussed on ledger meeting that's better to separate to small tasks. And the sending votes in bunch is not related to adding reward votes directly.
I will resolve conflicts later

Reply:
it is but only for independent things that wont create conflicts, which is not this case.